backend/domain/DTO/response/DiscussionEndDTO.js

```diff
@@ -1,6 +1,6 @@
 class DiscussionEndDTO {
-    constructor(playerKilled) {
-        this.playerKilled = playerKilled;
+    constructor(playersOnTrial) {
+        this.playersOnTrial = playersOnTrial;

```

Comment:
Nitpick: should be `playerOnTrial` right? Since we simplified the logic down.

Reply:
could we possibly leave it as playersOnTrial for future implementation? If not it should fine to get rid of it as well!

Reply:
Yeah, we could rename it for this instance as people could get confused while working on this :) If we do allow multiple players to go on trial, we can rename the variable later!